### DISCUSSION TITLE
Likes instead of "thank you" comments / Лайки вместо благодарных комментариев

### DISCUSSION THREAD

**Discussion by yuriy-zhilovets2** [PastVu/pastvu#636] [Ideas / Идеи и предложения]
Ввести "Спасибо", которым можно благодарить за привязку фото. Во-первых, комментарии не будут засоряться вежливыми, но не очень нужными словами. Во-вторых, можно вести статистику и рейтинг лучших следопытов.

[Mot](https://pastvu.com/u/Mot)
- возможно речь не только о привязке но и о доп.инфо. Вероятно, Вы предлагаете значки "лайки". Я бы тоже ставил значок советчику в знак благодарности (или согласия с ним) за любую ценную инфо к моему фото. И чтоб это не занимало отдельную строку, а стоял в первой строке, где ник, и после даты.

**Comment by paul-k-pastvu** [comment]
Согласен, отличная идея. Разовью.
Социальные реакции нужны в  целом ряде случаев.
1) Просто как благодарность за тот или иной комментарий
2) Как оценка того или иного действия (которое может и не сопровождаться комментарием)
При этом обсуждение привязки или датировки того или иного снимка может вестись даже не под этим снимком.

В связи с этим, нужно:
3) библиотека доступных "социальных реакций" - "спасибо". "спасибо за геопривязку", "спасибо за датировку", "согласен", "не согласен", ... (текстовые обозначения выводятся в виде всплывающей подсказки на том языке интерфейса, который установлен у пользователя)
4) значки для каждой реакции
5) возможность для участников маркировать такими значками действия или комментарии других участников

При этом:
6) Пометить реакцией "спасибо за геопривязку" и "спасибо за датировку" можно комментарий под снимком (но не в новости)
7) Такую пометку может сделать только участник, загрузивший снимок, или модератор (администратор).
8) можно пометить действия в истории снимка, при этом пометка будет относиться к тому участнику, который выполнил это действие. 
9) себя самого (и свои действия или комментарии) пометить нельзя

В профиле пользователя может отображаться статистика по полученным благодарностям от других участников (а также по тем, которые он сам оставил)

**Comment by paul-k-pastvu** [comment]
См. также #333
